docker/docker_config.sh

```diff
@@ -5,6 +5,13 @@ DOCKER_IMAGE_NAME="ghcr.io/lasp/adamant:example-latest"
 export DOCKER_CONTAINER_NAME
 export DOCKER_IMAGE_NAME
 
+case "$OSTYPE" in
+  linux-gnu)
```

Comment:
Instead of `linux-gnu`, let's try to handle more cases and make it `linux*`. Retest and make sure that still works. Inspiration: https://megamorf.gitlab.io/2021/05/08/detect-operating-system-in-shell-script/

Reply:
That sounds much better - see new commits reflecting this